libs/remix-ui/home-tab/src/lib/components/homeTabFeatured.tsx

```diff
@@ -32,33 +33,45 @@ function HomeTabFeatured() {
               sliderClass="h-100 justify-content-between"
               deviceType={"desktop"}
               itemClass=""
-              autoPlaySpeed={15000}
+              autoPlaySpeed={150000}
```

Comment:
remember to change it back before merging this :)

Reply:
oh yes. taking it back now